cloudant-client/src/main/java/com/cloudant/client/api/scheduler/SchedulerDocsResponse.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright © 2018 IBM Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+package com.cloudant.client.api.scheduler;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+public class SchedulerDocsResponse {
+
+    private List<Doc> docs;
+    private long offset;
+    @SerializedName("total_rows")
+    private long totalRows;
+
+    public SchedulerDocsResponse(List<Doc> docs, long offset, long totalRows) {
+        this.docs = docs;
+        this.offset = offset;
+        this.totalRows = totalRows;
+    }
+
+    public List<Doc> getDocs() {
+        return docs;
+    }
+
+    public long getOffset() {
+        return offset;
+    }
+
+    public long getTotalRows() {
+        return totalRows;
+    }
+
+    public class Doc {
+
+        public Doc(String database, String docId, long errorCount, String id, Object info,
```

Comment:
Is this constructor needed?

Reply:
It's not needed for deserialisation. I had thought it was needed to silence findbugs but just double-checked and it's not needed so I'll remove them.